admin.py

```diff
@@ -800,17 +800,41 @@ def BuildSensorList(self, software_version):
         recording |= 1
       if sensor.supports_min_max_recording:
         recording |= 2
+      try:
+          range_min=int(sensor.range_min)
+      except:
+          range_min = None
+      try:
+          range_max=int(sensor.range_max)
+      except:
+          range_max = None
+      try:
+          normal_min=int(sensor.normal_min)
+      except:
+          normal_min = None
+      try:
+          normal_max=int(sensor.normal_max)
+      except:
+          normal_max = None 
+      try:
+          unit=str(sensor.unit)
+      except:
+          unit = None 
 
       sensors.append({
         'description': str(sensor.description),
         'index': int(sensor.index),
         'supports_recording': recording,
         'type': int(sensor.type),
+        'range_min': range_min,
```

Comment:
TBH you could just do the int() bit here and not bother with the try/catch for now (depending on how we feel generally about parsing invalid/partial data), or does not doing it above break all "old" model collector data?